packages/synthetic-chain/src/cli/build.ts

```diff
@@ -62,21 +62,9 @@ export const buildProposalSubmissions = (proposals: ProposalInfo[]) => {
   }
 };
 
-export const buildTestImages = (proposals: ProposalInfo[], dry = false) => {
-  for (const proposal of proposals) {
-    if (!dry) {
-      console.log(
-        `\nBuilding test image for proposal ${proposal.proposalName}`,
-      );
-    }
-    const { name, target } = imageNameForProposal(proposal, 'test');
-    // 'load' to ensure the images are output to the Docker client. Seems to be necessary
-    // for the CI docker/build-push-action to re-use the cached stages.
-    const cmd = `docker buildx build --load --tag ${name} --target ${target} .`;
-    console.log(cmd);
-    if (!dry) {
-      // `time` to output how long each build takes
-      execSync(`time ${cmd}`, { stdio: 'inherit' });
-    }
-  }
+export const bakeImages = (matrix: 'test' | 'use', dry = false) => {
```

Comment:
`dry` is a bit of a misnomer in this function, since it always execs the `bake`.

```suggestion
export const bakeImages = (matrix: 'test' | 'use', verbose = false) => {
```

Reply:
The `--print` option is instead of building. I'll clarify.